### PR TITLE
WIP: Rahti4 migration

### PR DESCRIPTION
**Pull request by lvarin**
## Proposed changes

Rahti production docs:

https://csc-guide-preview.rahtiapp.fi/origin/Rahti4-migration/

Very big PR

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](../CONTRIBUTING.md) and [Styleguide](../STYLEGUIDE.md) documents.
- [ ] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list)?

### CODE REVIEW

File: docs/cloud/pouta/advanced.md
Comment (tolonenj) [inline docs/cloud/pouta/advanced.md:1]: This file will need content at some point, but first we should discuss / think of what to consider as 'advanced' level topics.

File: docs/cloud/pouta/configuration.md
Comment (tolonenj) [inline docs/cloud/pouta/configuration.md:1]: Same comments as with advanced.md file. This should be thought through. But don't stop merging imho.

**Review by tolonenj — APPROVED** [review]
Creating a virtual machine instructions of Pouta needs another pair of eyes. I detached the ssh key part from it and just linked it, but may need more thorough thought whether there are parts to be moved under own tutorial (and again just link to the vm article). Otherwise looked good. Naturally I'm biased to say so since made quite many of these changes.

**Comment by trispera** [comment]
NEW PR: https://github.com/CSCfi/csc-user-guide/pull/1920

**Pull request closed without merging**